agb/src/display/tiled/map.rs

```diff
@@ -359,13 +356,7 @@ impl AffineMap {
             crate::syscall::bg_affine_matrix(transform_origin.into(), self.scroll, scale, rotation);
     }
 
```

Comment:
probably worth adding a method to set the transform matrix manually from our own methods rather than having to use set_transform? Or replacing `set_transform` with one that takes the transform matrix and provide a method which allows you to create a transformation matrix with a given origin, scale and rotation?

Reply:
Would want to remove set_scroll_pos as it does nothing